IdokladSdk/Authentication/Tokenizer.cs

```diff
@@ -40,7 +46,20 @@ public Tokenizer()
         /// Gets access token.
         /// </summary>
         [JsonProperty("access_token")]
-        public string AccessToken { get; internal set; }
+        public string AccessToken
+        {
+            get => _accessToken;
+            internal set
+            {
+                _accessToken = value;
+                ParseAccessToken();
```

Comment:
Zvážit, že by se neparsovalo tady, ale místo _jwtToken by byla property a parsovalo by se v getteru.